CHANGELOG.md

```diff
@@ -1,5 +1,112 @@
 # Changelog
 
+## v1.9.0 (2020-8-6)
+
+### Enhancements
+
+#### 🌟 Multi Window Support (Beta)
+
+TestCafe can now automate user actions in multiple windows. You can switch between open windows during the test. Make edits in one window and check that the other window's content changes dynamically to reflect these modifications.
+
+![Testing in multiple windows](docs/articles/images/blog/2020-07-23-multi-window.gif)
+
+When the main window opens a child window, TestCafe automatically switches to this new window and continues test actions there:
+
+```js
+import { Selector } from 'testcafe';
+
+fixture `Login page`
+    .page('https://login.wrike.com/login/');
```

Comment:
Can we use the real third-party websites in our examples?

Reply:
AFAIR, we can't show them on screenshots and videos, because this could include their proprietary visuals. I'm not sure if mentioning a URL is bad. I asked @AndreyBelym

Reply:
I think it's better to avoid such situations if we can